salt/modules/dockerng.py

```diff
@@ -504,10 +504,14 @@ def __virtual__():
                 log.warning(
                     'Insufficient Docker version for dockerng (required: '
                     '{0}, installed: {1})'.format(
-                        '.'.join(docker_versioninfo),
-                        '.'.join(MIN_DOCKER)
-                    )
-                )
+                        '.'.join(map(str, MIN_DOCKER)),
+                        '.'.join(map(str, docker_versioninfo))))
+        else:
+            log.warning(
```

Comment:
@jfindlay 
That's an awesome hint!  I was trying to figure out how to get this done, but could not find any documentation on it, and grepping the existing modules only revealed `return False` everywhere I've looked.
Where should (or is) this documented?